.gitignore

```diff
@@ -28,3 +28,4 @@ pom.xml
 pom.xml.asc
 profiles.clj
 repl-port
+/tmp
```

Comment:
maybe we can use `git-integration-testing`, so that the intent is clearer and don't end up storing unrelated stuff under `tmp`